br/pkg/metautil/metafile.go

```diff
@@ -528,22 +528,14 @@ func (writer *MetaWriter) FinishWriteMetas(ctx context.Context, op AppendOp) err
 	var err error
 	// flush the buffered meta
 	if !writer.useV2Meta {
-		// Set schema version
-		writer.backupMeta.Version = MetaV1
-		err = writer.flushMetasV1(ctx, op)
+		writer.flushMetasV1(ctx, op)
```

Comment:
How about rename flushMetasV1 to another name since we don't flush here anymore?>

Reply:
Got it, do it later.